tools/scripts/gen-env-files.sh

```diff
@@ -352,8 +352,18 @@ KSQL_PRODUCER_INTERCEPTOR_CLASSES='io.confluent.monitoring.clients.interceptor.M
 KSQL_CONSUMER_INTERCEPTOR_CLASSES='io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor'
 KSQL_KSQL_CONNECT_URL='http://connect:8083'
 KSQL_KSQL_LOGGING_PROCESSING_TOPIC_REPLICATION_FACTOR=1
-KSQL_KSQL_LOGGING_PROCESSING_TOPIC_AUTO_CREATE='true'
```

Comment:
Latest version of Kafka has a bug where it attempts to create clustered logging topics.  With a single node it fails infinitely...